### ISSUE TITLE
Rationalize top-level Web API pages

### ISSUE THREAD

**Issue by wbamberg** [mdn/content#1579]
According to  https://github.com/mdn/sprints/issues/3327 we have ~1100 pages directly under Web/API.

I believe that every page directly under Web/API must be one of:

1) an API overview page, e.g. https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API
2) an interface page, e.g. https://developer.mozilla.org/en-US/docs/Web/API/Request
3) a dictionary page, e.g. https://developer.mozilla.org/en-US/docs/Web/API/AesGcmParams 

While it's hard to tell which pages are in group (1), a reasonable heuristic seems to be: if the title contains a space, the page is an API overview page. But that heuristic we have about 100 API overview pages, leaving about 1000 others.

As far as I know, all the others ought to be either interface pages or dictionary pages, so ought to be listed in [GroupData.json](https://github.com/mdn/kumascript/blob/master/macros/GroupData.json). In fact, ~250 of these pages are not listed in GroupData.

Additionally, some interfaces/dictionaries appear under more than one group in GroupData. I believe that this should not be allowed.

So we should:

* [ ] tag API overview pages, so we know exactly which ones they have
* [ ] ensure all API overview pages have a corresponding entry in GroupData.json
* [ ] tag all dictionary pages
* [ ] ensure all dictionary pages are listed under an entry in GroupData.json
* [ ] tag all interface pages
* [ ] ensure all interface pages are listed under an entry in GroupData.json
* [ ] relocate or delete any other pages that aren't in one of these three categories
* [ ] remove any duplicate interface/dictionary entries from GroupData

**Comment by sideshowbarker** [comment]
As this one is nearing 2 years without any updated, I suggest we consider closing it and maybe instead perhaps adding to the OWD work list (if it’s not already included there) — following the general principle that it’s good to not keep issues open indefinitely without any associated updates to the content.

**Comment by wbamberg** [comment]
Fair enough. I still think this is worth doing but it's obsolete at least in that it talks about using tags rather than page types, and it might (I hope) be obsoleted by https://github.com/openwebdocs/project/issues/76, so I'm less invested in getting it done.
